x-pack/plugins/upgrade_assistant/public/application/components/overview/upgrade_step/upgrade_step.tsx

```diff
@@ -7,119 +7,33 @@
 
 import React from 'react';
 
-import {
-  EuiText,
-  EuiFlexItem,
-  EuiFlexGroup,
-  EuiSpacer,
-  EuiButton,
-  EuiButtonEmpty,
-} from '@elastic/eui';
+import { EuiText, EuiSpacer } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
 import type { EuiStepProps } from '@elastic/eui/src/components/steps/step';
-import { useAppContext } from '../../../app_context';
 
 const i18nTexts = {
   upgradeStepTitle: i18n.translate('xpack.upgradeAssistant.overview.upgradeStepTitle', {
-    defaultMessage: 'Upgrade to Elastic 8.x',
+    defaultMessage: 'Get ready for the next version of Elastic',
   }),
   upgradeStepDescription: i18n.translate('xpack.upgradeAssistant.overview.upgradeStepDescription', {
     defaultMessage:
-      'Once you’ve resolved all critical issues and verified that your applications are ready, you can upgrade to Elastic 8.x. Be sure to back up your data again before upgrading.',
+      'You can start resolving all critical issues in preparation for the upcoming Elastic 8.x. Be sure to back up your data again before upgrading',
```

Comment:
Here you used "in preparation" 😊 It could also be "...to prepare for".

Reply:
The current copy is missing a period:
```suggestion
      'You can start resolving all critical issues in preparation for the upcoming Elastic 8.x. Be sure to back up your data again before upgrading.',
```

This seemed a little odd to me as it came _after_ step 2. At this point, the user should already have solved their critical issues, right? 

```suggestion
      'If you've resolved all critical issues and checked your deprecation logs, you're ready for the next major version of Elastic. Check this page again when you upgrade to Elastic 8.x.',
```

<img width="1230" alt="Screen Shot 2021-11-29 at 10 38 51 AM" src="https://user-images.githubusercontent.com/40268737/143897548-7e6fecbe-4743-4675-b790-b70ea16613ff.PNG">

Reply:
++ to James's second suggestion. "Be sure to back up your data again before upgrading" seems out of place, given that we don't expect the user to upgrade to 8 from 7.16. "Check this page again..." guides the user to the correct next step in the process.